OrbitGl/TimeGraph.cpp

```diff
@@ -697,6 +763,41 @@ void TimeGraph::OnRight() {
   NeedsUpdate();
 }
 
+void TimeGraph::SelectAndZoom(const TextBox* text_box) {
+  if (!text_box) return;
```

Comment:
Can this be a CHECK?

Reply:
Done! Had to modify all calls to SelectAndZoom only if it isn't NULL.